test/index.spec.ts

```diff
@@ -1,12 +1,57 @@
 import { newUnibeautify, Beautifier } from "unibeautify";
-import beautifier from "../src";
+import { beautifier, fixType, application } from "../src";
 test("should successfully install beautifier", () => {
   const unibeautify = newUnibeautify();
   unibeautify.loadBeautifier(beautifier);
   expect(unibeautify.loadedBeautifiers.map(curr => curr.name)).toEqual([
     beautifier.name,
   ]);
 });
+test("should modify options to appropriate data type", () => {
+  const options = {
+    wrap: "asdf",
+  };
+  fixType(options, { wrap: 0 });
+  expect(options.wrap).toEqual(0);
+});
+test("should validate prettydiff.language.auto is a function", () => {
+  const unibeautify = newUnibeautify();
+  unibeautify.loadBeautifier(beautifier);
+  const text = `function test(n){return n+1;}`;
+  return unibeautify
+    .beautify({
+      languageName: "JavaScript",
+      options: {
+        JavaScript: {
+          indent_char: " ",
+          indent_size: 2,
+        },
+      },
+      text,
+    })
+    .then(() => {
+      expect(typeof application.api.language.auto).toBe("function");
```

Comment:
@prettydiff No need to export `application` as an alias for `prettydiff`. Just import `prettydiff` in the tests.

Reply:
Please review again.